evaluator/builtin_time.go

```diff
@@ -390,6 +390,16 @@ func builtinCurrentTime(args []types.Datum, _ context.Context) (d types.Datum, e
 	return convertToDuration(d, fsp)
 }
 
+// See https://dev.mysql.com/doc/refman/5.7/en/date-and-time-functions.html#function_utc-date
+func builtinUTCDate(args []types.Datum, _ context.Context) (d types.Datum, err error) {
+	year, month, day := time.Now().Date()
+	t := mysql.Time{
+		Time: time.Date(year, month, day, 0, 0, 0, 0, time.Local),
```

Comment:
https://github.com/pingcap/tidb/blob/master/util/types/datum.go#L621

Probably, we need a new `case mysql.TypeDate:`